azure-pipelines.yml

```
@@ -308,9 +308,8 @@ stages:
               PYTEST+="--network=host --mount type=bind,source=${PWD}/script,target=/script "
               PYTEST+="greatexpectations/test:${DOCKER_TAG} /bin/bash -c \""
               PYTEST+="mysql --host=localhost --protocol=TCP --port=3306 --user=root --password='' --reconnect < /script/mysql_setup.sql && "
-              PYTEST+="pytest --mysql --random-order --ignore=tests/cli --ignore=tests/integration/usage_statistics "
               # The last \" closes the quote from bash -c \".
-              PYTEST+="--ignore=tests/rule_based_profiler/data_assistant/test_onboarding_data_assistant.py\""
+              PYTEST+="pytest --mysql --random-order tests/test_definitions tests/expectations\"
```

```suggestion
              PYTEST+="pytest --mysql --random-order tests/test_definitions tests/expectations\""
```

You are missing the "extra" `"` at the end of this line. The first one cloaes the `"` started on line 309, `bash -c \""`